lib/utils/index.js

```diff
@@ -489,7 +489,7 @@ class Utils {
   static stripControlChars(input) {
     return input && input.replace(
       // eslint-disable-next-line no-control-regex
-      /[\x00-\x09\x0B-\x0C\x0E-\x1F\x7F-\x9F]/g,
+      /[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/g,
```

Comment:
I was mistaken here, sorry. This shouldn't be changed, as it is being used in a few places already. What we need is actually the strip-ansi package: https://www.npmjs.com/package/strip-ansi